addons/sourcemod/scripting/nt_classlimit.sp

```diff
@@ -122,11 +122,40 @@ public void OnPluginStart()
 		HookClassSelectionPfns(client);
 		break;
 	}
-
+	
+	RegAdminCmd("sm_classlimit", Command_Limit, ADMFLAG_GENERIC);
+	
 	// Create the default config file, if it doesn't exist yet
 	AutoExecConfig();
 }
 
+public Action Command_Limit(int client, int args)
+{
+	if (args == 0)
+	{
+		ReplyToCommand(client, "Usage !classlimit x where x sets the limit for all classes");
+		return Plugin_Handled;
+	}
+	
+	char limitArg[2 + 1];
+	GetCmdArg(1, limitArg, sizeof(limitArg));
+	int limit = StringToInt(limitArg);
+
+	if(limit == 0)
+	{
+		limit = 16;
+		ReplyToCommand(client, "Setting limit failed, setting limits to 16 each");
+	}
+
+	g_Cvar_MaxRecons.SetInt(limit);
+	g_Cvar_MaxAssaults.SetInt(limit);
+	g_Cvar_MaxSupports.SetInt(limit);
+	
+	PrintToChatAll("Class limit set to %d each", limit);
+	
```

Comment:
Alternatively, you could add the `FCVAR_NOTIFY` flag to the cvars, and use the `notify` parameter of `ConVar.SetInt` for printing the change automatically.

Reply:
Yeah this might be good if the command allowed changing individual limits, maybe it's just simpler for now to leave it as one line printout.